dev_guide/copy_files_to_container.adoc

```diff
@@ -95,15 +95,15 @@ are not in the local directory.
 [[continuous-syncing-on-file-change]]
 == Continuous Syncing on File Change
 
-The `--watch` flag causes the command to monitor the source path for file system
-changes and sync changes each time they occur. With this argument, the command
-will run forever.
+Using the `--watch` option causes the command to monitor the source path for any
+file system changes and syncronizes changes when they occur. With this argument,
+the command will run forever.
```

Comment:
do you need "will"? Could you just say: 
With this argument, the command runs forever
?